android/res/layout/show_on_map_fab.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<com.google.android.material.floatingactionbutton.FloatingActionButton
+  xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
+  android:id="@+id/fabViewOnMap"
```

Comment:
Why it fabViewOnMap instead of fab_view_on_map? Or even better, to match the layout, show_on_map_fab? Then it will be very easy to search/replace in the future.

Reply:
I copied the original fab from `fragment_bookmark_list.xml` so I did not think about changing the name but yes it would make more sense